.github/workflows/frontend.yml

```diff
@@ -40,11 +40,20 @@ jobs:
         username: ${{ secrets.AWS_ACCESS_KEY_ID }}
         password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
-    - name: Build and push
+    - name: Build and push AMD
       uses: docker/build-push-action@v2
       with:
         context: ./services/frontend
-        platforms: linux/amd64,linux/arm64
+        platforms: linux/amd64
         push: true
-        tags: ${{ secrets.PUBLIC_ECR_REGISTRY }}/storedog/frontend:latest
+        tags: ${{ secrets.PUBLIC_ECR_REGISTRY }}/storedog/frontend-amd:latest
```

Comment:
We'd need to tag these as `storedog/frontend:amd-latest` and `storedog/frontend:arm-latest`